src/scanners/javascript.js

```diff
@@ -81,7 +63,8 @@ export default class JavaScriptScanner {
         ecmaVersion: ECMA_VERSION,
         sourceType: this.sourceType,
       },
-      rules: _ruleMapping,
+      rules,
+      rulePaths: [path.join(__dirname, '..', 'rules', 'javascript')],
```

Comment:
Yeah, if we want to adopt this approach, we'll have to double check that part. I haven't checked the build step yet because I am not too afraid of it, but it's likely broken indeed.